sessions/testing.qmd

```diff
@@ -113,9 +113,10 @@ These checks verify that the package has:
 
 among other things.
 
-To-Do: Explain the output of a CRAN check
+TODO: Explain the output of a CRAN check
+
+## Exercise: Create your first unit test
```

Comment:
Ideally, we would go through creating a unit test first, and then they do it again on their own time. So it could be something like "Write a unit test to check for ABC". The exact details don't matter too much, as long as there is a general structure or steps.

Reply:
This first exercise could even be something like "Review these example unit tests and identify the individual parts within and find out if they work as intended or not". So it could be more of a code reading exercise.